src/man/firejail-profile.txt

```diff
@@ -269,9 +278,13 @@ blacklist-nolog /usr/bin/gcc*
 .TP
 \fBbind directory1,directory2
 Mount-bind directory1 on top of directory2. This option is only available when running as root.
+Directories will retain the ownership and permissions of the original directory being mounted over. (directory2)
```

Comment:
Suggestion:
Directories will retain the ownership and permissions of the original directory being mounted over (directory2).
After termination, modifications affect the overlay directory (directory1).